common/src/main/java/com/wynntils/models/items/annotators/gui/GambitAnnotator.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright © Wynntils 2025.
+ * This file is released under LGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.models.items.annotators.gui;
+
+import com.wynntils.core.text.StyledText;
+import com.wynntils.handlers.item.GuiItemAnnotator;
+import com.wynntils.handlers.item.ItemAnnotation;
+import com.wynntils.models.items.items.gui.GambitItem;
+import com.wynntils.utils.colors.CustomColor;
+import com.wynntils.utils.mc.LoreUtils;
+import java.awt.Color;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import net.minecraft.world.item.ItemStack;
+
+public final class GambitAnnotator implements GuiItemAnnotator {
+    private static final Pattern NAME_PATTERN = Pattern.compile("^§(#[0-9A-Fa-f]{6,8})§l(.+? Gambit)$");
+    private static final Pattern ASPECT_PULLS_PATTERN = Pattern.compile("^§b- §7Aspect Pulls: §f(\\d+)$");
+    private static final Pattern REWARD_PULLS_PATTERN = Pattern.compile("^§b- §7Reward Pulls: §f(\\d+)$");
+
+    @Override
+    public ItemAnnotation getAnnotation(ItemStack itemStack, StyledText name) {
+        Matcher matcher = name.getMatcher(NAME_PATTERN);
+        if (!matcher.matches()) return null;
+        CustomColor color = CustomColor.fromHexString(matcher.group(1));
+        String itemName = matcher.group(2);
+
+        Optional<Integer> startIndex = getRewardTitleStartLine(LoreUtils.getLore(itemStack));
```

Comment:
`LoreUtils#getLore` can be expensive. Save the result, so you don't have to call it twice (even though this usage might get removed by my other suggestion).